tests/qmltests/tst_Shell.qml

```diff
@@ -3246,5 +3246,93 @@ Rectangle {
             tryCompare(priv, "currentItem", menuItem0);
             tryCompare(priv.currentItem, "popupVisible", true);
         }
+
+        function test_launcherLockedWhenNoAppsRunning_data() {
+            return [{
+                tag: "phone",
+                shell: "phone",
+                formFactor: "phone",
+                launcherLocked: false,
+                stateWithOpenIndicators: "",
+                stateWithOpenApps: ""
+            },
+            {
+                tag: "tablet",
+                shell: "tablet",
+                formFactor: "tablet",
+                launcherLocked: false,
+                stateWithOpenIndicators: "visible",
+                stateWithOpenApps: ""
+            },
+            {
+                tag: "desktop without auto-hide",
+                formFactor: "desktop",
+                shell: "desktop",
+                launcherLocked: true,
+                stateWithOpenIndicators: "visible",
+                stateWithOpenApps: "visible"
+            },
+            {
+                tag: "desktop with auto-hide",
+                formFactor: "desktop",
+                shell: "desktop",
+                launcherLocked: false,
+                stateWithOpenIndicators: "visible",
+                stateWithOpenApps: ""
+            },
+            {
+                tag: "desktop with auto-hide at small formFactor",
+                formFactor: "phone",
+                shell: "desktop",
+                launcherLocked: false,
+                stateWithOpenIndicators: "",
+                stateWithOpenApps: ""
+            },
+            {
+                tag: "tablet with auto-hide at small formFactor",
+                formFactor: "phone",
+                shell: "tablet",
+                launcherLocked: false,
+                stateWithOpenIndicators: "",
+                stateWithOpenApps: ""
+            },
```

Comment:
Missing closing bracket for the list of objects.

Reply:
Queue "how did this even work for me before" response.